projects/go-lib/src/styles/_mixins.scss

```diff
@@ -17,3 +17,34 @@ $z-index: (
 @function z-index($key) {
   @return map-get($z-index, $key);
 }
+
+@mixin fill-text-background($fill: null, $color: $base-dark) {
+  // sass-lint:disable-block no-vendor-prefixes property-sort-order
```

Comment:
This reminds me, we need to get a sass lint configuration implemented into goponents. I'll make an issue for that.